src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -577,4 +577,7 @@ In order to debug this project, add an executable project to this solution which
     <value>An error occurred during a design-time build. Some features may not work correctly. See the Error List for details.</value>
     <comment>"Error List" is the title of the Visual Studio tool window that error messages are listed in.</comment>
   </data>
+  <data name="HotReloadRebuildFail" xml:space="preserve">
+    <value>HotReload rebuild fail</value>
+  </data>
```

Comment:
In the branching, "Hot Reload" is two words. As above, this is a build, not a rebuild.

Also, is the user notified of the build failure via any other means?

```suggestion
  <data name="HotReloadBuildFailed" xml:space="preserve">
    <value>Hot Reload build failed</value>
  </data>
```

Should the message provide the user with an explanation of what to do? For example "Fix the build issue and try again."

Reply:
Or perhaps even a longer message like "Unable to restart the project due to a build failure. Fix the build and try again."

Given this resource is only used in the restart case, maybe make the name specific to that too. Something like `HotReloadRestartBuildFailed`